model/trace/messaging.yaml

```diff
@@ -180,3 +180,6 @@ groups:
         tag: tech-specific-gcp-pubsub
         requirement_level:
           conditionally_required: If the message type has an ordering key set.
+      - ref: messaging.gcp_pubsub.message.link
```

Comment:
Is "link" a terminology already used in GCP for this specific use case? As a non GCP experienced person, I find just "link" a bit confused after reading the description. If it's not a well-defined term already, perhaps we can try to find a more clear one? WDYT?

Reply:
Decided to remove the link attribute. I think it's confusing people more than helping.